src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/VisualizationCard/VisualizationCard.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../../GenericDelegateCard';
+import LegendItem from './LegendItem';
+import { RelativeDelegateBar } from './RelativeDelegateBar';
+
+interface Props {
+  percent: number;
+  totalDai: number;
+}
+
+const VisualizationCard: React.FC<Props> = ({ percent }) => (
+  <ExtendedKeyStatsCard>
+    <Legend>
+      <LegendItem color="#ECF1F3" description="Other Expenses" />
+      <LegendItem color="#447AFB" description="Recognized Delegates (21)" />
+    </Legend>
+    <ContainerBar>
+      <BarPercent>
+        <RelativeDelegateBar otherExpenses={90} recognizedDelegates={15} />
```

Comment:
For me, here you will need 2 props `otherExpenses` to know the number of expenses of the core units and other elements except delegates, and `delegatesExpenses` which is the number of the delegate's expenses, with that info you can calculate the totalDAI and pass down to the component to calculate the percentage, this way you will have a component more ready to integration later

Reply:
done